cluster/gce/config-default.sh

```diff
@@ -95,6 +95,11 @@ if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
   TARGET_NODE_UTILIZATION="${KUBE_TARGET_NODE_UTILIZATION:-0.7}"
 fi
 
+# Optional: Create autoscaler for cluster's pods.
```

Comment:
"cluster's pods" suggests this is for some special infrastructure pods. Maybe:

```Enable feature for autoscaling number of pods```

Otherwise people may think that there is one global autoscaler that you create, which will affect all pods. WDYT?